ddtrace/tracer/stats.go

```diff
@@ -164,9 +170,28 @@ func (c *concentrator) Stop() {
 	}
 	close(c.stop)
 	c.wg.Wait()
+empty:
```

Comment:
These are generally called bluntly `loop`, but in this case I think a self-descriptive better name would be `drain`.